app/models/event.rb

```diff
@@ -27,19 +27,4 @@ class Event < ApplicationRecord
   belongs_to :project
 
   validates :name, :data, presence: true
-  validates :handleable, presence: true, if: proc { |event| event.handled_type? }
-  validates_with EventNameValidator, if: proc { |event| handleable && event.handled_type? }
-  validates :occurred_at, presence: true, if: proc { |event| event.handled_type? }
```

Comment:
why are we removing validations?

Reply:
The first validation is in the github service already, we are duplicating code and making the validation twice.
The second one I believe is not necessary at all (correct me if I'm wrong)
The third one, the same as the first one.

Reply:
@mrubi-rootstrap might know better. Regarding the first and third validation, I get your point but remember that if in the future there is another piece of code creating/updating events then we'll be missing these validations

Reply:
So would it be better to validate it only in the model? I would like to keep those only in one place.

Reply:
ideally we would do business validations outside models and integrity validations in the models. What do you think these 2 are business or integrity validations?

Reply:
I believe is most likely to be a business validation rather than an integrity validation.